### ISSUE TITLE
Samsung QE75Q80T Home app icon turns to off immediately after using the accessory

### ISSUE THREAD

**Issue by tzclucian** [tavicu/homebridge-samsung-tizen#417]
Hello @tavicu 

Running 5.1.0. 
The TV powers on for couple of seconds then in Homekit the TV icon immediately shows it's off altough the TV is on.

I've tried starting from scratch and even updating the accessories config file by myself but this keeps happening.

Any hints?

PS: My api info includes a powerstate: on field. Model: QE75Q80TATXXH

I also get this logs when restarting Homebridge

Starting to advertise 'Homebridge E0FC 869C' using bonjour-hap backend!
[10/6/2021, 1:05:43 PM] Homebridge v1.3.4 (Homebridge E0FC) is running on port 51449.
Preparing Advertiser for 'TV 9D48' using bonjour-hap backend!
Starting to advertise 'TV 9D48' using bonjour-hap backend!
[10/6/2021, 1:05:43 PM] TV 9D48 is running on port 56244.
[10/6/2021, 1:05:43 PM] Please add [TV 9D48] manually in Home app. Setup Code: 936-45-292
[10/6/2021, 1:06:15 PM] [TV] TV is offline

**Comment by tavicu** [comment]
Everything you posted is ok until the last line.
Does that line appears when you click to make an action or just appears on the console?

**Comment by tzclucian** [comment]
Nope, it simply appears after a while no matter whether I use the TV or the Home app.

**Comment by tzclucian** [comment]
Done.

Yes, that's why it seemed weird to me as well.

The TV is paired. I've clicked allow and all the settings under `Device Connection Manager` are set to allowed.

Access Notifications is set to `off`.

```
Preparing Advertiser for 'Living Room TV 9D48' using bonjour-hap backend!
Starting to advertise 'Living Room TV 9D48' using bonjour-hap backend!
[10/6/2021, 1:15:46 PM] Living Room TV 9D48 is running on port 56788.
[10/6/2021, 1:15:46 PM] Please add [Living Room TV 9D48] manually in Home app. Setup Code: 936-45-292
[10/6/2021, 1:16:00 PM] [Living Room TV] [DEBUG] TV is offline
[10/6/2021, 1:16:05 PM] [Living Room TV] [DEBUG] TV is offline
[10/6/2021, 1:16:10 PM] [Living Room TV] [DEBUG] TV is offline
[10/6/2021, 1:16:10 PM] [Living Room TV] Failed to pair! Make sure TV is online and you click "Allow" on the popup
[10/6/2021, 1:16:10 PM] [Living Room TV] [DEBUG] PairFailedError: Failed to pair! Make sure TV is online and you click "Allow" on the popup
```

**Comment by tavicu** [comment]
Are you able to open this url? `http://TV_IP:8001/api/v2/`, replace TV_IP with your ip from config

Everything you are showing me indicates that the homebridge server can't access the IP address of TV.

1. Pair-ing is failing cause the plugin can't reach the IP
2. Can't get the installing apps because of the same problem
3. Can't send commands cause the plugin can't reach socket of TV on that IP address

**Comment by tzclucian** [comment]
I'm a programmer myself so that should be ok.

I'll reply soon with the updates.

Thank you so much for your help.

**Comment by tavicu** [comment]
Ok, so ping it's ok but the request to http is taking too long and gets cancelled.
The timeout is set to 250ms and it's configurable.

Let's change it to 1500ms and see the results.
Example how to change it:
```
{
    "platform": "SamsungTizen",
    "devices": [
        {
            "name": "Bedroom TV",
            "ip": "10.20.30.40",
            "mac": "A0:B1:C2:D3:E4:F5",
            "timeout": 1500
        }
    ]
}
```

After you change it restart homebridge instance.

**Comment by tzclucian** [comment]
It started to work much better but the same error still happens from time to time. Should I further increase the timeout?

**Comment by tavicu** [comment]
Try to hard reboot your TV by pressing power button on the remote until the samsung logo appears.
Wait for a few minutes (2-3) for TV to settle (it's responding very hard at the beginning until it loads everything) then restart the homebridge server.

That request should open in a few ms. So increasing the timeout it's not an option cause it will delay everything in the flow.

**Comment by tavicu** [comment]
I knew for sure i already followed all the steps to diagnose this in another issue. This is the issue: https://github.com/tavicu/homebridge-samsung-tizen/issues/324

You could look over there especially starting with this comment: https://github.com/tavicu/homebridge-samsung-tizen/issues/324#issuecomment-804746398

As you could se the request should take ~7ms to respond.

In the end his problem was with the machine it was running homebridge on and not the TV itself.

**Comment by tavicu** [comment]
Awesome. Glad to see it's ok now :)
